### PR TITLE
Added theme and navbar component

### PR DESCRIPTION
**Pull request by narharikale**
following feat. added 
- dark theme
- light theme 
- navbar

### CODE REVIEW

File: src/frontend/components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+.searchbar input{
+    height: 2rem;
+    flex-grow: 1;
+}
+
+
+@media screen and (max-width: 40em){
+
+
+}
```
Comment (znjs) [inline src/frontend/components/navbar/navbar.css:108]: can remove the empty media query

File: src/frontend/components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+@media  screen and (min-width: 64.063em) and (max-width: 90em) {
+    .searchbar-container{
+        display: flex;
+    }
+}
+
+
+@media  screen  and (max-width: 120em) {
+
+}
```
Comment (znjs) [inline src/frontend/components/navbar/navbar.css:125]: can remove the empty media query

**Review by znjs — APPROVED** [review]
- clean UI
- good naming conventions used
- easily understandable code

**Review by rahulmistry751 — APPROVED** [review]
Clean UI and code
Looks good to merge

**Pull request merged**